src/main/java/org/spongepowered/api/service/permission/Subject.java

```diff
@@ -80,6 +80,17 @@
 public interface Subject extends Contextual {
 
     /**
+     * Returns the friendly identifier associated with this Subject.
```

Comment:
What is considered friendly? Human readable? A player's name? An identifier stripped of all special characters?